salt/annotations/init.sls

```diff
@@ -8,19 +8,38 @@ annotations-nginx-vhost:
         - listen_in:
             - service: nginx-server-service
 
-annotations-repository:
-    builder.git_latest:
-        - name: git@github.com:elifesciences/annotations.git
-        - identity: {{ pillar.elife.projects_builder.key or '' }}
-        - rev: {{ salt['elife.rev']() }}
-        - branch: {{ salt['elife.branch']() }}
-        - target: /srv/annotations/
-        - force_fetch: True
-        - force_checkout: True
-        - force_reset: True
-        - fetch_pull_requests: True
-        - require:
-            - cmd: composer
+annotations-folder:
+    # carefully removing everything but config.php and var/logs
+    # which are mounted inside the container
```

Comment:
They're provisioned afterwards, so what's the concern? (Losing production logs?)

Reply:
Losing logs but also interrupting the service (especially if something goes wrong and the process is interrupted)

Reply:
Not ideal, but temporary. So, 🤷‍♂️ .